planck/PLKCommandLine.m

```diff
@@ -107,6 +107,7 @@ +(int)processArgsCount:(int)argc vector:(char * const *)argv
         {"repl", no_argument, NULL, 'r'},
         {"static-fns", no_argument, NULL, 's'},
         {"no-banner", no_argument, NULL, 'z'},
+        {"quiet", no_argument, NULL, 'q'},
```

Comment:
There is a string here that the `q` would also be added to: https://github.com/mfikes/planck/blob/1c4157eb7db4339df50a597e6e56477762715e71/planck/PLKCommandLine.m#L117